lib/HALSensorFusionTarget/SensorFusionProximitySensors.h

```diff
@@ -0,0 +1,139 @@
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+/*******************************************************************************
+    DESCRIPTION
+*******************************************************************************/
+/**
+ * @brief  Proximity sensors realization for the Sensor Fusion App
+ * @author Juliane Kerpe <juliane.kerpe@web.de>
+ *
+ * @addtogroup HALTarget
+ *
+ * @{
+ */
+#ifndef SENSORFUSIONPROXIMITYSENSORS_H
+#define SENSORFUSIONPROXIMITYSENSORS_H
+
+/******************************************************************************
+ * Compile Switches
+ *****************************************************************************/
+
+/******************************************************************************
+ * Includes
+ *****************************************************************************/
+#include <stdint.h>
+#include <IProximitySensors.h>
+
+/******************************************************************************
+ * Macros
+ *****************************************************************************/
+
+/******************************************************************************
+ * Types and Classes
+ *****************************************************************************/
+
+/** This class provides access to a dummy driver for the Proximity Sensors of the Sensor Fusion Application.  */
+class SensorFusionProximitySensors : public IProximitySensors
+{
+public:
+    /**
+     * Constructs the ProximitySensors adapter.
+     */
+    SensorFusionProximitySensors() : IProximitySensors()
+    {
+    }
+
+    /**
+     * Destroys the ProximitySensors adapter.
+     */
+    virtual ~SensorFusionProximitySensors()
+    {
+    }
+
+    /**
+     * Initialize only the front proximity sensor.
+     */
+    void initFrontSensor() final
+    {
+    }
+
+    /**
+     * Returns the number of sensors.
+     *
+     * @return Number of sensors
+     */
+    uint8_t getNumSensors() const final
+    {
+        return 1;
```

Comment:
Missing "U" postfix after unsigned interger:  `1U`
Please fix everywhere where relevant

Reply:
Corrected also in ProximitySensors of HALTest.